snippets/read-values-by-signal.md 

```diff
@@ -0,0 +1,28 @@
+---
+title: Reading values by signal
+tags:
+  - tips
```

Comment:
This is very cool, but probably belongs to 30 seconds of rxjs?

Reply:
@kirjs since, there's no such repo as 30secs of rxjs I suggest leaving it here

Reply:
We'll have one :)